### ISSUE TITLE
FACET_4_2 CSV files do not have any headings to describe their columns

### ISSUE THREAD

**Issue by e-remington-lee** [A2Zadeh/CMU-MultimodalSDK#239]
Hello!

I am trying to understand the dataset more but the FACET_4_2 csv files do not have any headers in them so I am extremely confused as to what each column is referring to. Here are the first 8 lines of the "_0efYOjQYRc.csv" file found in the CMU_MOSEI/Raw/Videos/Full/FACET_4_2.

0,0.0333333,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
1,0.0666667,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
2,0.1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
3,0.133333,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
4,0.166667,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
5,0.2,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
6,0.233333,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
7,0.266667,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,

**Comment by A2Zadeh** [comment]
Hello,

The headings are as follows: 

dimensions=["frame","timestamp","Anger","Contempt","Disgust","Joy","Fear","Baseline","Sadness","Surprise","Confusion","Frustration","AU1","AU2","AU4","AU5","AU6","AU7","AU9","AU10","AU12","AU14","AU15","AU17","AU18","AU20","AU23","AU24","AU25","AU26","AU28","AU43","Has_Glasses","Is_Male","Pitch","Yaw","Roll"]

This information is provided also in the metadata of the computational sequences.

**Comment by e-remington-lee** [comment]
Thank you very much!

**Comment by A2Zadeh** [comment]
@e-remington-lee More than happy to help! Good luck with your experiments. Feel free to ask other questions.

**Comment by e-remington-lee** [comment]
Hello @A2Zadeh!

I downloaded the raw MOSEI dataset. Where would I find the "metadata of computational sequences" you mentioned? I found the labels for the video facial expressions in FACET_4_2 folder, but where would I find the labels for the audio and transcript emotional intensity? I see there is a metadata folder, but it does not seem to have any computation sequences you mentioned. I downloaded the raw dataset and the folder structure is as follows. Thanks a lot in advance.

Audio
--Full
----COVAREP
----WAV_16000
Labels
--5 CSV files
Metadata
--Standard_test_fold
--Standard_train_fold
--standard_valid-fold
Transcripts
--Full
----Aligned
------P2FA Files
----Combined
------Transcript files
Videos
--Full
----OpenFace
----FACET
----Combined
--Segmented
